<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwECoGDialogUi</class>
 <widget class="QDialog" name="AwECoGDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECOG Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="1,0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select electrodes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Grids/Strips:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonGrid">
     <property name="text">
      <string>Add to GRID/Strip</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>376</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>616</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="buttonApply">
     <property name="text">
      <string>Add to Montage</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resources/AnyWave.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonApply</sender>
   <signal>clicked()</signal>
   <receiver>AwECoGDialogUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReset</sender>
   <signal>clicked()</signal>
   <receiver>AwECoGDialogUi</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reset()</slot>
 </slots>
</ui>
